### ISSUE TITLE
a issue about negative samples in logits

### ISSUE THREAD

**Issue by haofuly** [vita-epfl/social-nce#13]
Thanks for your great work!
https://github.com/vita-epfl/social-nce/blob/61630607c56b3a5a5d4ef5320aab15d10307ac3a/crowd_nav/snce/contrastive.py#L68
I am not sure why **"sim_neg.view(sim_neg.size(0), -1).repeat_interleave(self.horizon, dim=0)"** rather than **"sim_neg.view(sim_neg.size(0) * sim_neg.size(1), -1)"**. The number of negative samples corresponding to one positive sample should be **(N-1)\*8** or **H\*(N-1)\*8**?
